src/api/userApi.ts

```diff
@@ -9,6 +9,7 @@ import { setUser } from '../redux/actions/userActions';
 
 export const getCurrentUserAttributes = () => {
   return async (dispatch: Dispatch) => {
+    const loggedIn = localStorage.getItem('amplify-signin-with-hostedUI');
```

Comment:
man er ikke nødvendigvis logget inn om man har dette i LS, men man har ikke aktivt logget ut siden forrige innlogging. Man kan feks ha expired token, men hvor denne da fortsatt vil være true. Men regner med dette bare er for å ikke få opp "kunne ikke hente bruker" etter utlogging, så det blir det uansett rett der

Reply:
ja, det er sant. Ja, det var derfor jeg sjekket på dette, men vi burde nok utvide funksjonaliteten på dette etterhvert. Ref denne også: https://unit.atlassian.net/browse/NP-1150